docs/reference/commandline/dockerd.md

```diff
@@ -780,6 +785,113 @@ conditions the user can pass any size less then the backing fs size.
 $ sudo dockerd -s overlay2 --storage-opt overlay2.size=1G
 ```
 
+
+#### Windowsfilter options
+
+##### `size`
+
+Specifies the size to use when creating the sandbox which is used for containers.
+Defaults to 20G.
+
+###### Example
+
+```PowerShell
+C:\> dockerd --storage-opt size=40G
+```
+
+#### LCOW (Linux Containers on Windows) options
+
+##### `lcow.globalmode`
+
+Specifies whether the daemon instantiates utility VM instances as required 
+(recommended and default if omitted), or uses single global utility VM (better
+performance, but has security implications and not recommended for production
+deployments).
+
+###### Example
+
+```PowerShell
+C:\> dockerd --storage-opt lcow.globalmode=false
+```
+
+##### `lcow.kirdpath`
+
+Specifies the folder path to the location of a pair of kernel and initrd files
+used for booting a utility VM. Defaults to `%ProgramFiles%\Linux Containers`.
+
+###### Example
+
+```PowerShell
+C:\> dockerd --storage-opt lcow.kirdpath=c:\path\to\files
+```
+
+##### `lcow.kernel`
+
+Specifies the filename of a kernel file located in the `lcow.kirdpath` path.
+Defaults to `bootx64.efi`.
+
+###### Example
+
+```PowerShell
+C:\> dockerd --storage-opt lcow.kernel=kernel.efi
+```
+
+##### `lcow.initrd`
+
+Specifies the filename of an initrd file located in the `lcow.kirdpath` path.
+Defaults to `initrd.img`.
+
+###### Example
+
+```PowerShell
+C:\> dockerd --storage-opt lcow.initrd=myinitrd.img
+```
+
+##### `lcow.bootparameters`
+
+Specifies additional boot parameters for booting utility VMs when in kernel/
+initrd mode. Ignored if the utility VM is booting from VHD. These settings
+are kernel specific.
+
+###### Example
+
+```PowerShell
+C:\> dockerd --storage-opt "lcow.bootparameters='option=value'"
+```
+
+##### `lcow.vhdx`
+
+Specifies a custom VHDX to boot a utility VM, as an alternate to kernel
+and initrd booting. Defaults to `uvm.vhdx` under `lcow.kirdpath`.
+
+###### Example
+
+```PowerShell
+C:\> dockerd --storage-opt lcow.vhdx=custom.vhdx
+```
+
+##### `lcow.timeout`
+
+Specifies the timeout for utility VM operations in seconds. Defaults
+to 300.
+
+###### Example
+
+```PowerShell
+C:\> dockerd --storage-opt lcow.timeout=240
+```
+
+##### `lcow.sandboxsize`
+
+Specifies the size in GB to use when creating the sandbox which is used for
+containers. Defaults to 20. Cannot be less than 20.
```

Comment:
I'm documenting what's implemented. Plus you told me to prefix options with `lcow.` 😇